addons/project/models/project_project.py

```diff
@@ -687,10 +687,18 @@ def get_milestones(self):
         return {}
 
     def _get_profitability_labels(self):
-        return {}
+        return {
+            'other_purchase_costs': _lt('Vendor Bills'),
+            'other_revenues': _lt('Other Revenues'),
+            'other_costs': _lt('Other Costs'),
+        }
```

Comment:
Why does all this now need to be in the base project addon?

Reply:
Good point, it actually should now be in project_account. At some earlier stage of the development , everything was put inside the project module, but  we ended up having to create a new bridge module. This part should be in it.